request.go

```diff
@@ -9,6 +9,7 @@ import (
 type Request struct {
 	context context.Context
 	client  *Client
+	headers map[string]string
```

Comment:
A header can have more than one value. I suggest changing the type using the http package headers themselves:
```go
headers http.Header
```

Reply:
done